app/src/main/java/com/nervousfish/nervousfish/service_locator/EntryActivity.java

```diff
@@ -21,7 +21,8 @@
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        final IServiceLocator serviceLocator = ServiceLocatorCreator.createInstance();
+        final IServiceLocator serviceLocator = ServiceLocatorCreator.createInstance(
```

Comment:
After some reconsideration, I think it is better to do this on a separate branch so we can start using the database in the meantime 👍